<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KiviatInfo</class>
 <widget class="QWidget" name="KiviatInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="5">
    <widget class="QListWidget" name="lstDataSets"/>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QPushButton" name="cmdRemoveDataSet">
     <property name="text">
      <string>remove data set</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="cmdChangeColor">
     <property name="text">
      <string>change color</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="QTreeWidget" name="treeDataSetInfo">
     <column>
      <property name="text">
       <string>Axis Label</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>13</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QPushButton" name="cmdSetRoot">
     <property name="toolTip">
      <string>Computes the names and ranges of the Metric Attributes in this subtree only</string>
     </property>
     <property name="text">
      <string>Recollect Metrics</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3" colspan="2">
    <widget class="QPushButton" name="cmdConfigMetrics">
     <property name="text">
      <string>Configure Metrics</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
